### DISCUSSION TITLE
8. iv)

### DISCUSSION THREAD

**Discussion by DanielN2020** [VDI-CodING/codeING-main#33] [S1D0] [answered]
Füge folgendes zu .eslintignore hinzu:

node_modules
build

wie funktioniert das unter Windows?

**Comment by DanielN2020** [comment, marked as answer]
Hat sich erledigt - unter Visual Studio die Datei anwählen und im Code eintragen

**Reply by Vince2312** [reply]
Also hast du in Zeile 1 und 2 hinzugefügt? Bei mir erscheint eine leere Datei bei VS wenn ich .eslintignore öffne.

**Reply by KlausLex** [reply]
Genau, einfach `node_modules` & `build` in Zeilen 1 & 2 einfügen und mit ctrl+s abspeichern

`New-Item -ItemType file .eslintignore` erstellt eine leere Datei mit Namen `.eslintignore`.
Ist wie ein leeres Textdokument in Word, das erst befüllt werden will :)
